[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/technical-guides/checklist-providing-high-availability-with-fault-tolerance-or-load-balancing.md

|                             Configure the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] databases on a clustered [!INCLUDE[btsSQLServerNoVersion](../includes/btssqlservernoversion-md.md)] instance on a Windows server cluster.                             |                                                                                                                                                                                                                                                                                                       [High Availability for Databases](../technical-guides/high-availability-for-databases.md)                                                                                                                                                                                                                                                                                                       |

> |                                                    Configure tracking hosts for high availability. Verify that there are at least N+1 host instances with tracking enabled where N is the number of MessageBox databases in the BizTalk group.                                                     |                                                                                                                          To enable tracking for a host, select the option to **Allow Host Tracking** on the **Host Properties** dialog box available from the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] Administration console. For more information about setting host properties, see the topic [How to Modify Host Properties](https://go.microsoft.com/fwlink/?LinkId=154359).                                                                                                                          |

Reviewer: ```suggestion
|                                                    Configure tracking hosts for high availability. Verify that there are at least N+1 host instances with tracking enabled where N is the number of MessageBox databases in the BizTalk group.                                                     |                                                                                                                          To enable tracking for a host, select the option to **Allow Host Tracking** on the **Host Properties** dialog box available from the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] Administration console. For more information about setting host properties, see the topic [How to Modify Host Properties](https://go.microsoft.com/fwlink/?LinkId=154359).                                                                                                                          |
```
```suggestion
|                                                    Configure tracking hosts for high availability. Verify that there are at least N+1 host instances with tracking enabled where N is the number of MessageBox databases in the BizTalk group.                                                     |                                                                                                                          To enable tracking for a host, select the option to **Allow Host Tracking** on the **Host Properties** dialog box available from the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] Administration console. For more information about setting host properties, see the topic [Update Host Properties](https://go.microsoft.com/fwlink/?LinkId=154359).                                                                                                                          |
```

|                                Verify there is enough processing power and memory on the passive cluster nodes to enable multiple [!INCLUDE[btsSQLServerNoVersion](../includes/btssqlservernoversion-md.md)] instances to run simultaneously in failover scenarios.                                |                                                                                If two SQL instances regularly consume over 50 percent of the resources on individual cluster nodes, then when both instances fail over to a single node, each instance will incur degraded performance. Therefore, testing should be performed to ensure that a single cluster node will be able to handle the load until the failed node is brought back online. If a single node cannot handle the load of both SQL instances then consider adding additional cluster nodes to implement an Active/Active/Passive cluster topology.                                                                                 |

> | Implement a storage area network (SAN) to house the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] databases. **Note:**  If possible, configure the SAN disks using RAID 1+0 (a stripe of mirror sets) topology for maximum performance and high availability. |                                                                                                                                                                                                                                    In the “BizTalk Server Database Optimization white paper”, see the "Disk Infrastructure" section under "Performance Tuning".                                                                                                                                                                                                                                    |

Reviewer: ```suggestion
| Implement a storage area network (SAN) to house the [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] databases. **Note:**  If possible, configure the SAN disks using RAID 1+0 (a stripe of mirror sets) topology for maximum performance and high availability. |                                                                                                                                                                                                                                    For more information, see the ["BizTalk Server Database Optimization" whitepaper](/biztalk/technical-guides/optimizing-database-performance).                                                                                                                                                                                                                                    |
```

|                                                                                                    Use Windows Clustering to cluster the Enterprise Single Sign-On (SSO) master secret server.                                                                                                     | [High Availability for the Master Secret Server](../technical-guides/high-availability-for-the-master-secret-server.md) **Note:**  Do not cluster the SSO service on a computer running [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] unless you cluster SSO and a BizTalk host in the same cluster group. For more information about clustering the SSO service and a [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] host in the same cluster group see [How to Cluster SSO and a BizTalk Host in the Same Cluster Group](https://go.microsoft.com/fwlink/?LinkId=154367). |

> |                                                                                                                     Back up the Enterprise Single Sign-On (SSO) Master Secret.                                                                                                                     |                                                                                                                                                                                                                                                                               See [How to Back Up the Master Secret](https://go.microsoft.com/fwlink/?LinkID=151395).                                                                                                                                                                                                                                                                                |

Reviewer: ```suggestion
|                                                                                                                     Back up the Enterprise Single Sign-On (SSO) Master Secret.                                                                                                                     |                                                                                                                                                                                                                                                                               See [How to Back Up the Master Secret](/biztalk/core/how-to-back-up-the-master-secret).                                                                                                                                                                                                                                                                                |
```

> |                                           Configure the Internet Information Services (IIS) Web server for isolated host instances and the BAM Portal Web page to be highly available using Network Load Balancing (NLB) or other load balancing device.                                           |                                                                                                                                                                                                                                                             **For Windows Server 2008**: See [Network Load Balancing Deployment Guide](https://go.microsoft.com/fwlink/?LinkId=153139).                                                                                                                                                                                                                                                              |

Reviewer: ```suggestion
|                                           Configure the Internet Information Services (IIS) Web server for isolated host instances and the BAM Portal Web page to be highly available using Network Load Balancing (NLB) or other load balancing device.                                           |                                                                                                                                                                                                                                                             **For Windows Server 2008**: See [Network Load Balancing Deployment Guide](/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc754833(v=ws.10)).                                                                                                                                                                                                                                                              |
```
